uefi-test-runner/src/proto/network/snp.rs

```diff
@@ -107,16 +147,10 @@ pub fn test() {
         let mut buffer = [0u8; 1500];
 
         info!("Waiting for the reception");
-        if simple_network.receive(&mut buffer, None, None, None, None)
-            == Err(Status::NOT_READY.into())
-        {
-            boot::stall(Duration::from_secs(1));
```

Comment:
Are we sure this stall is no longer needed? I'm guessing it was added because sometimes the interface isn't ready yet, although I'm not clear on the details from a quick look at the commit history.